apps/ui/src/composables/useAccount.ts

```diff
@@ -9,22 +9,19 @@ watch(
   () => web3.value.account,
   (current, previous) => {
     if (current !== previous) {
+      votes.value = {};
       pendingVotes.value = {};
     }
   }
 );
 
 export function useAccount() {
   async function loadVotes(networkId: NetworkID, spaceIds: string[]) {
-    votes.value = {};
-
     const account = web3.value.account;
     if (!account) return;
 
     const network = getNetwork(networkId);
-    const userVotes = await network.api.loadUserVotes(spaceIds, account);
-
-    votes.value = { ...votes.value, ...userVotes };
+    votes.value = await network.api.loadUserVotes(spaceIds, account);
```

Comment:
Should we merge previous and current values (like we had before)?

Right now if I'm switching between two spaces and on one space there is proposal I already voted on then when I enter it will show "not voted" state for a while and then change to "already voted" state, before it would use already available data, now it will throw it away.

Reply:
https://github.com/user-attachments/assets/11244be9-ca0d-4822-b996-b37433fbf29f

Reply:
What you see is also happening on master.

The merging with the previous values never worked, as we were always clearing the whole object on each call (line 19 before this pr, now removed)

This PR was just removing the delay between when the array is cleared and repopulatedm but I guess we can also fix this other issue in this PR